src/providers/VernaProvider.tsx

```diff
@@ -202,14 +224,17 @@ function VernaProvider({
           fullUiSchema,
           handleSubmit,
           isEditor,
+          locale,
```

Comment:
Do not pass locale into the VernaProvider. This value is already stored through the IntlProvider. So if we you need to retrieve this value in a child, you can use `useIntl`